templates/details/libraries/libraries_layout.html

```diff
@@ -81,7 +81,23 @@ <h3>{{ docstrings["name"] }}</h3>
           <li class="p-inline-list__item">
             <ul class="p-inline-list has-vertical-separator">
               <li class="p-inline-list__item">
-                <a href="#" class="is-always-color-link"><i class="p-icon--download">Download</i> <span>Download</span></a>
+                <div class="p-modal-tooltip">
```

Comment:
Looks nice. Can we make it a bit bigger, so we can see all the command.

And click out of the modal should close it.

Reply:
If it goes bigger it will go offscreen to the right. The alternative would be to extend it to the left but that will disassociate it from the button.
 
> And click out of the modal should close it.
I'd agree if this was actually a modal but it's a panel related to that button. @danielmutis do you have any thoughts on these points?